var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,291 @@
+                        '.tf_configure.bazelrc')
+        if self.spec.satisfies('@1.13.1'):
+            # tensorflow_estimator is an API for tensorflow
+            # tensorflow-estimator imports tensorflow during build, so
+            # tensorflow has to be set up first
+            filter_file(r"'tensorflow_estimator >=",
+                        r"#'tensorflow_estimator >=",
+                        'tensorflow/tools/pip_package/setup.py')
+        if self.spec.satisfies('@2.0.0:'):
+            # now it depends on the nightly versions...
+            filter_file(r"'tf-estimator-nightly >=",
+                        r"#'tf-estimator-nightly >=",
+                        'tensorflow/tools/pip_package/setup.py')
+            filter_file(r"REQUIRED_PACKAGES\[i\] = 'tb-nightly >=",
+                        r"pass #REQUIRED_PACKAGES\[i\] = 'tb-nightly >=",
+                        'tensorflow/tools/pip_package/setup.py')
+            filter_file(r"'tb-nightly >=",
+                        r"#'tb-nightly >=",
+                        'tensorflow/tools/pip_package/setup.py')
+
+        if self.spec.satisfies('+nccl'):
+            filter_file(r'^build --action_env NCCL_INSTALL_PATH=.*',
+                     r'build --action_env NCCL_INSTALL_PATH="'+str(spec['nccl'].prefix.lib)+'"',
+                     '.tf_configure.bazelrc')
+            filter_file(r'^build --action_env NCCL_HDR_PATH=.*',
+                     r'build --action_env NCCL_HDR_PATH="'+str(spec['nccl'].prefix.include)+'"',
+                    '.tf_configure.bazelrc')
+
+        if self.spec.satisfies('+cuda'):
+            libs = [
+                    str(spec['cuda'].prefix)+'/lib',
+                    str(spec['cuda'].prefix)+'/lib64',
+                    str(spec['cudnn'].prefix)+'/lib',
+                    str(spec['cudnn'].prefix)+'/lib64',
+                    str(spec['nccl'].prefix)+'/lib',
+                    str(spec['nccl'].prefix)+'/lib64']
+            slibs = ':'.join(libs)
+
+            filter_file('build --action_env TF_NEED_OPENCL_SYCL="0"',
+                        'build --action_env TF_NEED_OPENCL_SYCL="0"\n'
+                        'build --action_env LD_LIBRARY_PATH="'+slibs+'"',
+                        '.tf_configure.bazelrc')
+
+
+        if '+cuda' in spec:
+            #TODO also consider the case '+gcp'
+            #TODO make '--cxxopt...CXX11_ABI=0' a variant
+            # For an explanation for ABI build option, see https://www.tensorflow.org/install/source#bazel_build_options
+            # Recently noticed that TF_NEED_AWS etc environment variables aren't recognized by configure.py anymore.
+            # So, explicitly disable them via bazel options.
+            bazel('build', '--jobs={}'.format(make_jobs), '-c', 'opt', '--config=cuda', '--config=noaws', '--config=nogcp','--config=nohdfs','--config=noignite','--config=nokafka', '--cxxopt=-D_GLIBCXX_USE_CXX11_ABI=0', '//tensorflow/tools/pip_package:build_pip_package')  # noqa: E501
```

Comment:
```suggestion
            bazel('build', '--jobs={0}'.format(make_jobs), '-c', 'opt', '--config=cuda', '--config=noaws', '--config=nogcp','--config=nohdfs','--config=noignite','--config=nokafka', '--cxxopt=-D_GLIBCXX_USE_CXX11_ABI=0', '//tensorflow/tools/pip_package:build_pip_package')  # noqa: E501
```
For Python 2.6 support. Also, no need for a single long line, you should get rid of all these `# noqa` and wrap lines properly.